include/seqan3/core/simd/view_to_simd.hpp

```diff
@@ -479,16 +473,16 @@ class view_to_simd<urng_t, simd_t>::iterator_type
     {
         size_t max_distance = 0;
         for (auto && [it, sent] : views::zip(iterators_before_update, cached_sentinel))
-            if (size_t distance = std::ranges::distance(it, sent); distance > max_distance)
-                max_distance = distance;
+            max_distance = std::max<size_t>(std::ranges::distance(it, sent), max_distance);
+
 
         assert(max_distance > 0);
         assert(max_distance <= (total_chunks * chunk_size));
 
         --max_distance;
         final_chunk_pos = max_distance  / chunk_size;
         // first we should be able to check the chunk position.
-        current_chunk_sizes[1] = (max_distance % chunk_size) + 1;
+        size_of_final_chunk = (max_distance % chunk_size) + 1;
```

Comment:
... and this `size_of_final_chunk`.

I think `final_chunk_size` is okay as name.